src/devices/devicemanager.cpp

```diff
@@ -348,6 +347,26 @@ DeviceInfo* DeviceManager::FindEquivalentDevice(DeviceInfo* info) const {
   return nullptr;
 }
 
+QList<DeviceInfo*> DeviceManager::FindDeviceByUrlSchemes(
```

Comment:
Minor: Should this be `FindDevicesByUrlSchemes`?

Reply:
done